AvaliadorDisciplinas.FRONTEND/src/pages/UserComentarios/index.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from "react";
+import styles from "./UserComentarios.module.css";
+import UserImage from "../../assets/icons/user_anonimous.svg";
+import NavMenu from "../../components/NavMenu";
+import { useAuth } from "../../contexts/AuthContext";
+import DataList from "../../components/DataList";
+import { getAll } from "../../services/comentariosService";
+import { renderItem } from "./comentariosListagem";
+import { useNavigate } from "react-router-dom";
+
+function DisciplinaIndex() {
```

Comment:
Nome do componente deveria ser `UserComentarios`.
```suggestion
function UserComentarios() {
```